cloudwash/providers/aws.py

```diff
@@ -60,13 +60,34 @@ def dry_pips():
                     [dry_data["PIPS"]["delete"].append(dpip["AllocationId"]) for dpip in rpips]
                 return rpips
 
+            def dry_stacks():
+                rstacks = []
+                if settings.aws.criteria.stacks.unassigned:
```

Comment:
You dont need this condition because we are not finding free stacks, we are just iterating over list of stacks. We should be deleting stacks those starts with the given text and has crossed SLA.